static/templates/default_language_modal.hbs

```diff
@@ -6,14 +6,20 @@
     </div>
     <div class="modal-body">
         <p>
+            {{#tr this}}
             The following is a few of the languages we have started to
             support or are hoping to support in the near future. We would
             love help translating; if you're interested in helping, make an
-            account at <a href='https://www.transifex.com/zulip/zulip/'>
-            https://www.transifex.com/zulip/zulip/</a>, and sign up for a
+            account at
+            {{/tr}}
+            <a href='https://www.transifex.com/zulip/zulip/'>
+                https://www.transifex.com/zulip/zulip/
+            </a>,
+            {{#tr this}} and sign up for a
             language! You can also request any language we don't currently
             have. It only takes a few hours to translate the most important
             parts of the app.
+            {{/tr}}
```

Comment:
You can't split a sentence in the middle this was because different languages have different word orders (etc.).  

@hackerkid can help but I think what we want to instead is pass the URL bit as a variable.

We may also want to pare down the text to something briefer about contributing translations, and have more or the text at the linked page (which we probably want to swap to our ReadTheDocs page on translating anyway), if the goal is to just to not have untranslated text in the app.